lib/stacks/iam.ts

```diff
@@ -73,9 +73,28 @@ export class IAMStack extends Stack {
         // Required tags that force the request to be specific to an MLSpace managed resource
         const requestTagsConditions = {
             'aws:RequestTag/project': 'false',
-            'aws:RequestTag/system': 'false',
+            // this is excluded because it is covered by requestSystemTagEqualsConditions below
```

Comment:
Do we need to have this commented out code here?

Reply:
I was trying to capture some knowledge so in the future it is obvious to anyone why `system` isn't part of these conditions without an in-depth knowledge of these permissions. If you think that is unnecessary (or worse, confusing) I am happy to remove it.

Reply:
You can leave it!